plugins/modules/scaleway_lb.py

```diff
@@ -113,47 +113,52 @@
     region: fr-par
 """
 
-RETURNS = """
-{
-   "scaleway_lb": {
-      "backend_count": 0,
-      "frontend_count": 0,
-      "description": "Description of my load-balancer",
-      "id": "00000000-0000-0000-0000-000000000000",
-      "instances": [
-         {
+RETURN = r"""
+scaleway_lb:
+  description: The load-balancer object.
+  returned: success
+  type: dict
+  sample:
+    {
+      "scaleway_lb": {
```

Comment:
I wonder whether the outer dict is actually there inside the `scaleway_lb` return value, or whether the contents of it are returned as `scaleway_lb`. @abarbare @remyleone do you know / can you test?

Reply:
oh, yup, I missed that. The content is the response from the Scaleway API, so I strongly suspect the answer is: we should remove that outer dict.

Reply:
Maybe @Mia-Cross (from the recent #10424 ) could help us confirm that?